int16 auto_man
int16 e-stop
int16 gear
int16 speed
int16 steer
int16 brake
float32 encoder